css/app/datepicker.css

```diff
@@ -89,27 +89,35 @@
 	border-left: 0;
 }
 
-#datepicker table tbody tr.uib-weeks td:last-child button {
-	border-right: none;
+.hilight-today {
```

Comment:
As mentioned previously by @jancborchardt - this should be `.highlight-today` (spelling)